cmd/background-controller/main.go

```diff
@@ -131,6 +120,7 @@ func main() {
 		internal.WithKubeconfig(),
 		internal.WithPolicyExceptions(),
 		internal.WithConfigMapCaching(),
+		internal.WithCosign(),
```

Comment:
I wonder if we need cosign in the background controller?

Reply:
It's not related to this change though.

@eddycharly mentioned the rule could reference image data from the context, and it makes sense to me.